test/unit/mocks.js

```diff
@@ -225,14 +226,20 @@ define('narrativeMocks', ['jquery', 'uuid', 'narrativeConfig'], ($, UUID, Config
      * - statusCode - int, default = 200 - the HTTP status code your request should return
      * - statusText - string, default = "HTTP/1.1 200 OK" - a status string your request will return
      * - response - object - the data your request should return as an object.
+     * - isError - boolean - if truthy, will format the JSON-RPC response with an error field
```

Comment:
Added an 'isError' toggle here - JSON-RPC errors get their response packet returned with an 'error' field instead of a 'result' field.